changed method for get discovery patient heading

DmitrySolyannik opened the pull request.

DmitrySolyannik committed:
changed method for get discovery patient heading

tony-shannon commented:
need more detail on this PR please

DmitrySolyannik commented:
@tony-shannon , I've understand that I am using wrong method for getting heading (getHeadingDetails), instead of I've used getHeadingSummary, in old version we had those 2 methods , but in new version in discovery module I have only one

DmitrySolyannik committed:
added configs for unit tests

killmenot commented on helm/discovery_service/spec/mocks/context.js:
```diff
@@ -34,14 +34,19 @@ const { ExecutionContext, QewdCacheAdapter } = require('../../lib/core');
 const WorkerMock = require('./worker');
 const CacheRegistryMock = require('./cache');
 const DbRegistryMock = require('./db');
+const globalConfig = require('../support/global_config.json');
```
please use `support/configuration.json` filename

DmitrySolyannik committed:
added configs for unit tests

DmitrySolyannik commented:
In this PR I've fixed few unit tests after config changes, changed code style (errors from lint), changed file name for config, also fixed one bug with getting discovery patient heading data

DmitrySolyannik commented:
Tested on mango-cookie and local environment, looks and works fine. I think we can merge this PR
